lib/exercism/submission.rb

```diff
@@ -103,6 +103,7 @@ def like!(user)
   def unlike!(user)
     likes.where(user_id: user.id).destroy_all
     self.is_liked = liked_by.length > 0
+    notifications.unread.where(action: 'like', actor_id: user.id).destroy_all
```

Comment:
Also I suspect this line will delete ALL unread like notifications, not just the one that applies to this submission.